### PR TITLE
Update Skip Updating preference strings.

### LINKED ISSUE #6699: Renaming skip updating feature
### Describe your suggested feature

Naming changes, to make it easier to understand

**Skip updating for:**
_Titles with unread chapters
Titles not being read
Titles with complete status_

Note 1: 1st option overwrite the 2nd one, so turn both on must not be possible
Note 2: I believe the repetition of "Titles..." is necessary to emphasize how each option works.

### Other details

[Begin of this idea](https://discord.com/channels/349436576037732353/694092459839455243/945051208882733066)

Another version:
**Do not update**
Titles with unread chapters 
Not started titles 
Completed titles_

A menu in this template for the first two options would be clearer (since both could not be activated at the same time)
image

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have updated the app to version **[0.13.1](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by FourTOne5**
In dialog -
```
Skip updating titles
- With unread chapter(s)
- That haven't been started
- With "Completed" status
```

In summary -
```
Skip updating titles
With unread chapter, With "Completed" status
```
Closes #6699

Co-authored-by: arkon <4098258+arkon@users.noreply.github.com>
Co-authored-by: Marco <76596575+SrW0nka@users.noreply.github.com>
Co-authored-by: specterflare <61708534+specterflare@users.noreply.github.com>

### CODE REVIEW

**Comment by specterflare** [comment]
Shouldn't it be "that _haven't_ been started"?

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -225,10 +225,10 @@
     <string name="charging">Charging</string>
     <string name="restrictions">Restrictions: %s</string>
 
-    <string name="pref_library_update_manga_restriction">Skip updating</string>
-    <string name="pref_update_only_completely_read">Has unread chapters</string>
-    <string name="pref_update_only_non_completed">Is completed series</string>
-    <string name="pref_update_only_started">No read chapters</string>
+    <string name="pref_library_update_manga_restriction">Skip updating titles</string>
+    <string name="pref_update_only_completely_read">with unread chapter</string>
```
Comment (arkon) [inline app/src/main/res/values/strings.xml:229]:
```suggestion
    <string name="pref_update_only_completely_read">With unread chapter(s)</string>
```

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -225,10 +225,10 @@
...
+    <string name="pref_update_only_non_completed">with \'Completed\' status</string>
```
Comment (arkon) [inline app/src/main/res/values/strings.xml:230]:
```suggestion
    <string name="pref_update_only_non_completed">With \"Completed\" status</string>
```

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -225,10 +225,10 @@
...
+    <string name="pref_update_only_started">that haven\'t been started</string>
```
Comment (arkon) [inline app/src/main/res/values/strings.xml:231]:
```suggestion
    <string name="pref_update_only_started">That haven\'t been started</string>
```

**Review by arkon — CHANGES_REQUESTED** [review]

**Pull request merged**